src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -450,8 +450,17 @@ private OpenApiOperation CreateBaseOperation(string description, List<OpenApiTag
 
             if (dbObject.SourceType == EntitySourceType.StoredProcedure)
             {
-                Entity entityTest = _runtimeConfig.Entities[entityName];
-                List<SupportedHttpVerb> spRestMethods = entityTest.Rest.Methods.ToList();
+                Entity entity = _runtimeConfig.Entities[entityName];
+
+                List<SupportedHttpVerb>? spRestMethods;
+                if (entity.Rest.Methods is not null)
+                {
+                    spRestMethods = entity.Rest.Methods.ToList();
```

Comment:
Is it possibe for rest to be disabled here?

Reply:
No. The caller of this method checks if rest is enabled for the entity before calling this method.

The check happens at `OpenApiDocumentor.BuildPaths()` : L 180 -186